Main.cpp

```diff
@@ -140,20 +140,22 @@ enum StarSize {
 
 struct Star {
     // Bitmap representation
-    BITMAP *m_pBitmap;
+    BITMAP *m_pBitmap = nullptr;
     // Center location on screen
-    Vector m_Pos;
-    // Bitmap offset
-//    int m_Offset;
+	std::array<int, 2> m_Pos = { 0 };
     // Scrolling ratio
-    float m_ScrollRatio;
-    // Normalized intensity 0-1.0
-    float m_Intensity;
+    float m_ScrollRatio = 1.0F;
+    int m_Intensity = 0;
     // Type
-    StarSize m_Size;
-
-    Star() { m_pBitmap = 0; m_Pos.Reset(); m_ScrollRatio = 1.0; m_Intensity = 1.0; m_Size = StarSmall; }
-    Star(BITMAP *pBitmap, Vector &pos, float scrollRatio, float intensity) { m_pBitmap = pBitmap; m_Pos = pos; m_ScrollRatio = scrollRatio; m_Intensity = intensity; }
+    StarSize m_Size = StarSmall;
+
+    Star() {}
+	Star(BITMAP* pBitmap, const std::array<int, 2>& pos, float scrollRatio, int intensity)
```

Comment:
Seems like spacing screwed up here. I guess one is tabs, the other is spaces? Best to make it all tabs.